src/main/java/it/hospital/project/appusers/UserService.java

```diff
@@ -0,0 +1,34 @@
+package it.hospital.project.appusers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class UserService {
+
+    UserRepository userRepository;
+
+    @Autowired
+    UserService(UserRepository userRepository) {
+        this.userRepository = userRepository;
+    }
+
+
+    public void addUser(User user) {
+        userRepository.save(user);
+    }
+
+    public void deleteUser(int id) {
```

Comment:
If the method is not implemented yet, why should we declare it?

Reply:
Defining an unimplemented method is fine if the development of this method is still in-progress.
Just make sure to put a TODO comment with supported explanation.

e.g.:

public void deleteUser(int id) {
      // TODO: will implement deleting of a user from the database
}

Reply:
According to YAGNI (extreme programming)  you should add functionalities only when you really need it. At this moment, we don't need this method (we have not specified endpoints in our controller) so I do not see any reasons to add it. 10 or 20 TODOs in code doesn't make it clearer.